lib/transformer/caster/to_atlas_attribute.rb

```diff
@@ -1,34 +1,29 @@
 module Transformer
   module Caster
-    class ToAtlasAttribute < Base
-      def analyze
-        static.merge(area_attributes).merge(init: initializer_inputs)
+    class ToAtlasAttribute
+      def self.filter(dataset_cast, template)
+        new(dataset_cast, template).filter
       end
 
-      private
-
-      def initializer_inputs
-        Atlas::InitializerInput.all.each_with_object({}) do |input, object|
-          if value = @analyzed_attributes[input.key]
-            object[input.key] = value
-          end
-        end
+      def filter
+        { area: area_attributes,
+          graph_values: @template.get(:graph_values) }
       end
 
-      def static
-        {
-          number_of_cars: number_of_cars,
-          number_of_residences: number_of_residences,
-          number_of_inhabitants: number_of_inhabitants,
-          number_of_old_residences: number_of_old_residences,
-          number_of_new_residences: number_of_new_residences,
-          has_industry: @dataset_edits.fetch(:has_industry),
-          has_agriculture: @dataset_edits.fetch(:has_agriculture)
-        }
+      private
+
+      def initialize(dataset_cast, template)
+        @dataset_cast = dataset_cast
+        @template     = template
       end
 
       def area_attributes
-        @analyzed_attributes.slice(*Atlas::Dataset.attribute_set.map(&:name))
+        @template.get(:area).each_pair do |area_key, value|
+          @dataset_cast.public_send("#{ area_key }=", value)
```

Comment:
I'm not opposed to `public_send("#{ area_key }=")` but we need to be 100% sure that `area_key` is a value we can trust (i.e. not something an end-user can specify without us validating it first). It looks fine at the moment – area data is only set in Caster::AreaAttributes – but... something to be aware of for the future.